cmd/ci-secret-bootstrap/main.go

```diff
@@ -223,6 +244,43 @@ func (o *options) validateCompletedOptions() error {
 	return nil
 }
 
+func constructDockerConfigJSON(bwClient bitwarden.Client, dockerConfigJSONData []secretbootstrap.DockerConfigJSONData) ([]byte, error) {
```

Comment:
This needs a very specific secret type, where do we set that?

Reply:
nice catch. We have already a way to add the type in the `to` config. I updated also the description.

Reply:
I will add validation for that.